### PR TITLE
r-fingerprint

### PR DESCRIPTION
**Pull request by eledant**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-fingerprint/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  # This is required to make R link correctly on Linux.
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+# Suggests: RUnit
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
```
Comment (lecorguille) [inline recipes/r-fingerprint/meta.yaml]: You need to remove all `[win]` lines

File: recipes/r-fingerprint/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+source:
+  fn: fingerprint_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/fingerprint_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/fingerprint/fingerprint_{{ version }}.tar.gz
+
+
+  sha256: ba7777c3b4a4cbd9acb2fc660660e55f0d49287060d2c9e27d00f210118a2de2
+  # patches:
```
Comment (lecorguille) [inline recipes/r-fingerprint/meta.yaml]: Please remove all useless comment lines

**Comment by lecorguille** [comment]
Please remove the `.bat` file as the lint procedure [suggest](https://bioconda.github.io/linting.html#has-windows-bat-file)

**Comment by daler** [comment]
Hi @eledant -- `fingerprint` looks like it is not specific to biology. If that's true, can you please add it to conda-forge instead? https://conda-forge.org/docs/

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**